pkgs/development/libraries/zeroc-ice/default.nix

```diff
@@ -95,9 +73,7 @@ in stdenv.mkDerivation rec {
     homepage = "https://www.zeroc.com/ice.html";
     description = "The internet communications engine";
     license = licenses.gpl2Only;
-    platforms = platforms.unix;
+    platforms = platforms.linux;
```

Comment:
I would leave `platforms.unix` here and mark macOS as broken instead.

Reply:
Otherwise you're limiting it for FreeBSD or some other sane systems that I assume this package may build.

Reply:
OK, done.